full/entrypoint.sh

```diff
@@ -10,4 +10,8 @@ credentials "${TF_HOSTNAME}" {
 EOF
 fi
 
+if [ -n "${KUBECONFIG_NAME}" ] && [ ! -f /root/.kube/config ] ; then
+  aws eks --region "${AWS_DEFAULT_REGION}" update-kubeconfig --name "${KUBECONFIG_NAME}"
```

Comment:
this is cool but you will do login on every single kubectl command, maybe we should add a better way for not repeating it every time

Reply:
Only if the container is removed. 
Because this if test the file config exist. 
But really in te new container the login will be do again.

Reply:
I can't think of anything now.